docs/modules/ROOT/pages/explanations/decisions/subscription-tracking.adoc

```diff
@@ -0,0 +1,71 @@
+= Subscription tracking
+
+== Problem
+
+We need to track the yearly subscriptions bought for OpenShift 4 clusters.
+Subscription count is required for a monthly report to RedHat.
+RedHat subscriptions are based on (v)CPU count.
+We need to cover the difference of the yearly subscription count and the actual CPU count with monthly subscriptions.
+
+Buying yearly subscriptions saves us money compared to buying monthly subscriptions.
+
+It also adds a small amount of risk if a customer decides to cancel their clusters before the year is over.
+
+=== Goals
+
+* Yearly subscriptions are tracked in a central place
+* Subscription count is available for an automated monthly report to RedHat
+* A suggestion for how many yearly subscriptions to buy is available to the account manager
+
+=== Non-goals
+
+* Full automation of subscription buying
+
+== Proposals
+
+=== Option 1: Database and Web UI (custom/ Grafana)
+
+We store subscriptions in a database and build a custom UI or Grafana dashboard to visualize the data.
+
+This allows instant feedback and would allow account managers to track subscriptions themselves without having to ask our team.
+
+Building the UI would take time and we would need to maintain it.
+
+Grafana most likely doesn't support editing data.
```

Comment:
I'd rather see a clear yes/no, rather than a "most likely" in a proposal. Either it can do it, or not.

Reply:
Editing data is possible. But creating a dashboard that comfortably works is not that easy.

Reply:
Got it. Are you able to describe it that way in the option?